### DISCUSSION TITLE
Check schema syntax and validity according to GraphQL-specification

### DISCUSSION THREAD

**Discussion by TaProhm** [kamilkisiela/graphql-inspector#2385] [General]
Is it possible to validate schema itself with Inspector ? It would be good to know that schema syntax is correct and schema is valid according to the spec. For example, that all interfaces are fully implemented etc.

I checked all available commands and didn't find any suitable functionality. 

Any help is appreciated.

**Comment by Urigo** [comment]
Hi @TaProhm , do you think that maybe [graphql-eslint](https://the-guild.dev/graphql/eslint) can give you that functionality?
